src/operators/implementation/onnx/operator__onnx__conv__11__T_tensor_float.c

```diff
@@ -2,162 +2,331 @@
 #include <stdlib.h>
 #include <string.h>
 #include <math.h>
+#include <stdbool.h>
 #include "trace.h"
 #include "utils.h"
+#include "tracing.h"
+#include "index.h"
 
 operator_status operator__onnx__conv__11__T_tensor_float(
     node_context *ctx
 )
 {
-  TRACE_LEVEL0("Calling operator_conv\n");
-
-  Onnx__TensorProto *X = searchInputByName(ctx, 0);
-  Onnx__TensorProto *W = searchInputByName(ctx, 1);
-  Onnx__TensorProto *B = searchInputByName(ctx, 2);
-
-  Onnx__TensorProto *Y = searchOutputByName(ctx, 0);
-
-  if (0){
-    /* TODO: Check some conditions. For example if a specific
-     * functionality is not supported */
-    //a->data_type == b->data_type
-    //a->n_dims == b->n_dims
-    //a->dims[i] == b->dims[i]
-    // dilations is hardcoded ?
-    fprintf(stderr, "Not implemented\n");
-    exit(-1);
+  TRACE_ENTRY(1);
+
+  TRACE_NODE(2, true, ctx->onnx_node);
+
+  Onnx__TensorProto *t_X = searchInputByName(ctx, 0);
```

Comment:
I liked having the same names as onnx has, `X`, `W`, so you can recognise the variable on a quick glance

Reply:
I had a naming problem with the attributes.
Since all attributes are optional, I couldn't use them directly.
I prefixed the onnx attributes with `a_` so I can refer to the actual values without the prefix.
`a_kernel` is the onnx attribute, `kernel` the usable value.
To be consistent I also added a tensor prefix `t_`.